pages/index/index.js

```diff
@@ -52,6 +55,62 @@ const GettingStarted = glamorous(Button)(
   }),
 )
 
+const StyledLiveProvider = glamorous(LiveProvider)({
+  margin: '0 auto',
+  display: 'flex',
+  flexDirection: 'column',
+  flexWrap: 'wrap',
+  justifyContent: 'space-around',
+  maxWidth: '50rem',
+})
+
+const StyledLiveEditor = glamorous(LiveEditor)({})
```

Comment:
If we're not going to apply styles, then I think we can just leave it as-is and just use the plain old `<LiveEditor />` below. Are we doing this in the `code-preview.js` file as well? 🙃